src/css.js

```diff
@@ -113,6 +113,14 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( window.top !== window.self && document.msFullscreenElement &&
+		elem.getBoundingClientRect().width > elem.offsetWidth ) {
```

Comment:
I have a Windows 10 VM around, I'll check if the problem exists in IE11 & Edge and to what extent.